mParticle-Primer/MPKitPrimer.h

```diff
@@ -20,7 +20,7 @@
 #import "MPKitExecStatus.h"
 #import "MPKitProtocol.h"
```

Comment:
Can you please replace these two lines with the following snippet:

```
#if defined(__has_include) && __has_include(<mParticle_Apple_SDK/mParticle.h>)
#import <mParticle_Apple_SDK/mParticle.h>
#else
#import "mParticle.h"
#endif
```

This should allow the import to work regardless of `use_frameworks!` or `use_frameworks!(false)` in the Podfile.